src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -3229,6 +3229,7 @@ internal static void SetSessionStateDrive(ExecutionContext context, bool setLoca
 
                         try
                         {
+                            providerContext.SuppressWildcardExpansion = true;
```

Comment:
(style) Maybe it's just my OCD, by why configuring providerContext inside try block rather than right after initialization?

Reply:
I wanted the two operations to be visually close, putting the assignment in the block did that for me